packages/react-core/src/components/Page/PageSection.tsx

```diff
@@ -56,17 +54,15 @@ export const PageSection: React.FunctionComponent<PageSectionProps> = ({
   children,
   variant = 'default',
   type = 'default',
-  hasNoPadding = false,
-  breakpointMods = [] as PageSectionBreakpointMod[],
+  padding,
   isFilled,
   ...props
 }: PageSectionProps) => (
   <section
     {...props}
     className={css(
       variantType[type],
-      hasNoPadding && styles.modifiers.noPadding,
-      formatBreakpointMods(breakpointMods, styles),
+      formatBreakpointMods(padding, styles),
```

Comment:
can you change the name of this to describe what it is formatting.

Reply:
Do you mean change the name of the function or of the prop `padding`?